qna/DE-MW-Ausfall.md

```diff
@@ -0,0 +1,17 @@
+## Question
+
+Was passiert wenn die fiskaltrust.Middleware nicht erreichbar ist?
+
+## Metadata tags
+
+lang-de, market-de, middleware, PosCreator, PosDealer, Consultant
+
+## Answer
+
+Bei einem Ausfall der fiskaltrust.Middleware darf mit der Kasse weiter gearbeitet werden (siehe Punkt 7 des AEAO zu § 146a: https://www.bundesfinanzministerium.de/Content/DE/FAQ/2020-02-18-steuergerechtigkeit-belegpflicht.html)
+
+Die TSE ist in dem Fall ebenso nicht erreichbar.
+
+- Die Kasse kann im „Ausfallmodus“ weiter betrieben werden.
+- Durch die Nacherfassung der ausgefallenen Belege kann können alle Daten, die während das Ausfalls von der Kasse gespeichert wurde, an die ft.MW gesendet werden, sobald diese wieder erreichbar ist. 
```

Comment:
- Typo: "kann können"
- Könntest du noch beschreiben wie das geht, oder link zu einer Beschreibung?

Reply:
Auch in diesem Fall kann der Hinweis auf die Vollständigkeit durch die Nacherfassung ergänzt werden.
"- Durch die Nacherfassung können alle gesetzlich erforderlichen Funktionen wie z.B. DSFinV-K-Export, PosArchiv, AKO, usw. dennoch erfüllt werden.
Die Behandlung des Ausfalls wird in der Interface-Dokumentation in https://docs.fiskaltust.cloud beschrieben."

Siehe auch https://fiskaltrust.de/fiskaltrust/faq/ - "Was passiert, wenn die Kasse die ft.Middleware nicht erreichen kann?"

Reply:
feedback wurde eingearbeitet (siehe neue commits)

Reply:
- der link ist falsch "https://docs.fiskaltust.cloud" -> da fehlt ein "r". Außerdem geht er nicht konkret auf die gewünschte stelle.
- der letzte Satz macht wenig Sinn, da es sich hier um die Übernahme exact dieser Frage aus FAQ DE mit dem exact gleichen Inhalt handelt. Vermutlich ist das einfach nur ein Hinweis von Axel.

ich habe beides korrigiert